### ISSUE TITLE
No gathered files 

### ISSUE THREAD

**Issue by amandengue** [rvdginste/todo-teamcity-plugin#1]
Hello

I'm just trying your plugin on my windows tc instance but its not able to retrieve file for analysis
My working directory is not empty at this time and contains this folder and files
Essimo.Logging
  Factory.cs
 Tools.cs
  version.cs

You can see my build step settings in the following logs.
Any idea?

> [19:20:47]Step 4/4: Open tasks (Todo Build Runner)
> [19:20:48][Step 4/4] The working root path is [D:\TeamCityBuildAgent\work\fdb2d0de0a1e9b54].
> [19:20:48][Step 4/4] The reporting root path is [D:\TeamCityBuildAgent\temp\buildTmp].
> [19:20:48][Step 4/4] Create directory for reporting [D:\TeamCityBuildAgent\temp\buildTmp\org.r4d5.teamcity.todo.report].
> [19:20:48][Step 4/4] Root path is [D:\TeamCityBuildAgent\work\fdb2d0de0a1e9b54].
> [19:20:48][Step 4/4] Included is [**\*. cs].
> [19:20:48][Step 4/4] Excluded is [**\*. config].
> [19:20:48][Step 4/4] Start gathering the files to be checked.
> [19:20:48][Step 4/4] Gathered 0 files.
> [19:20:48][Step 4/4] Minors include [// TODO.*].
> [19:20:48][Step 4/4] Majors include [// HACK.*].
> [19:20:48][Step 4/4] Criticals include [// BUG.*].
> [19:20:48][Step 4/4] Storing TodoScanResults in [D:\TeamCityBuildAgent\temp\buildTmp\org.r4d5.teamcity.todo.report\TodoScannerResults.ser]

**Comment by rvdginste** [comment]
@amandengue 

I think it either has something to do with the globbing pattern, or something to do with the directory separators. The backend will create a pattern like:

```
D:\TeamCityBuildAgent\work\fdb2d0de0a1e9b54\**\*.cs
```

And I'm not 100% sure it will match with:

```
D:\TeamCityBuildAgent\work\fdb2d0de0a1e9b54\Factory.cs
```

So, you could try to add the pattern '*.cs' to see if that works.

I always tested with a TeamCity running on Linux. If adding the extra pattern didn't help, I will try to replicate this on Windows.

I see your patterns for minor/major/critical and I'd like to mention that those patterns are case sensitive and really must match the whole line, so you'll probably need to add something like `.*` or `\s*` in front of those. I was a bit torn between the choice for more control or more convenience, and in the end chose for more control.

**Comment by amandengue** [comment]
@rvdginste 

Thank you for the reply
I had already tried the '*.cs' pattern without success.

**Comment by rvdginste** [comment]
@amandengue 

Small update: I reproduced the problem on Windows and I know where it goes wrong. It is Windows specific. I hope to have time this weekend to release a fix for it.

**Comment by rvdginste** [comment]
@amandengue 

This bug is fixed in the new release 1.0.1.
